packages/odyssey/src/scss/abstracts/_tokens.scss

```diff
@@ -12,7 +12,7 @@
 
 // Typography
 $body-font-family: 'Public Sans', '-apple-system',  'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', sans-serif;
-$mono-font-family: 'Inconsolata', 'SFMono-Regular', Consolas, 'Liberation Mono', Menlo, Courier, monospace;
+$mono-font-family: 'Inconsolata', 'SFMono-Regular', consolas, 'Liberation Mono', menlo, courier, monospace;
```

Comment:
curious: is the case meaningful for fonts? I would assume it was but unsure on the changes here - also curious about quoted single word fonts like `'Inconsolata'` for example

Reply:
I think this was an auto-fix change due to some of these typefaces not being quoted.

I'll lookup ideal behavior today and update or revert this.

Reply:
Looks like both quoted and unquoted syntaxes are acceptable by browsers today. CSS identifiers (`sans-serif`) will fail if they are quoted, however.

The rec from sass-guidelin.es is to quote all strings that are not identifiers, just to make things easy and uniform.

For font-family specifically, we could also declare our own `'system'` font that references local system font files in the future, avoiding these long declarations inline.